trickler/scales.py

```diff
@@ -344,11 +344,91 @@ def _stable_unstable(self, line):
         self.resolution = self.resolution_map[self.unit]
         # Update memcache values.
         self._update_memcache()
+        
+        
+class USSolidScale(SerialScale):
+    """Class for controlling a U.S. Solid USS-DBS81-110G scale."""
+
+    def __init__(self, config, port='/dev/ttyUSB0', baudrate=9600, timeout=0.1, **kwargs):
```

Comment:
Is 9600 the correct maximum baudrate of your scale?

Reply:
That is correct.